environment.yml

```diff
@@ -1,4 +1,4 @@
-name: quantecon
+name: lecture-python-programming
```

Comment:
let's revert this as we have a universal `quantecon` environment now.

Reply:
oops niceeeee. okay will do.